front/components/sparkle/AppLayout.tsx

```diff
@@ -160,6 +163,7 @@ export default function AppLayout({
   user,
   owner,
   topNavigationCurrent,
+  topNavigationChildren,
```

Comment:
I think this is too much complexity as we can likely get away without it. Additionally it’s likely that we’ll get rid to some extent of subNavigation as we converge towards the figma proposals so I’m reluctant to add something that is the thing above the subNavigation.

In a world without subNavigation then this collapses with the other children hinting at the fact that we should not have it imho.

Reply:
Yes agreed this adds more complexity, the only reason I did that is to have the "New Conversation Button" on top of the subNavigation, as in the designs.